src/recommendpolicy/recommendPolicy.go

```diff
@@ -136,31 +136,63 @@ func RecommendPolicyMain() {
 		return
 	}
 	systempolicy.InitSysPolicyDiscoveryConfiguration()
+	admissioncontrollerpolicy.InitAdmissionControllerPolicyDiscoveryConfiguration()
 	for _, d := range deployments.Items {
 		deploy := uniqueNsDeploy(d.Name, d.Namespace)
 
 		if deploy != nil {
 			DeployNsName = append(DeployNsName, *deploy)
 		}
 
-		for _, ns := range nsNotFilter {
+		for _, ns := range nsNotFilterSysPolicy {
 			if d.Namespace != ns {
 				generateHardenPolicy(d.Name, d.Namespace, d.Spec.Template.Labels)
 			}
 		}
+
+		nsNotFilterAdmissionControllerPolicy := cfg.CurrentCfg.ConfigAdmissionControllerPolicy.NsNotFilter
+		nsFilterAdmissionControllerPolicy := cfg.CurrentCfg.ConfigAdmissionControllerPolicy.NsFilter
+		recommendAdmissionControllerPolicy := cfg.GetCfgRecommendAdmissionControllerPolicy()
+
+		if recommendAdmissionControllerPolicy {
+			if len(nsFilterAdmissionControllerPolicy) > 0 {
+				for _, ns := range nsFilterAdmissionControllerPolicy {
+					if d.Namespace == ns {
+						generateAdmissionControllerPolicy(d.Name, d.Namespace, d.Spec.Template.Labels)
+					}
+				}
+			} else if len(nsNotFilterAdmissionControllerPolicy) > 0 {
+				for _, ns := range nsNotFilterAdmissionControllerPolicy {
+					if d.Namespace != ns {
+						generateAdmissionControllerPolicy(d.Name, d.Namespace, d.Spec.Template.Labels)
+					}
+				}
+			} else {
+				generateAdmissionControllerPolicy(d.Name, d.Namespace, d.Spec.Template.Labels)
+			}
+		}
```

Comment:
Please refactor this code on following lines...
```suggestion
	    if recommendAdmissionControllerPolicy && !isNsFiltered(d.Namespac) {
			generateAdmissionControllerPolicy(d.Name, d.Namespace, d.Spec.Template.Labels)
		}
```
implement `isNsFiltered()` to check if the namespace is filtered.

Reply:
Done.